example-app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,7 @@
     <string name="merchant_account_key">merchant_account</string>
     <string name="merchant_account_title">Merchant Account</string>
     <string name="additional_data_title">Additional Data</string>
+    <string name="app_title">App</string>
```

Comment:
hmm it's not important, but can we change this `app_title` to something like `app_title_key` because it's confusing.